Packs/SymantecCloudSOC/Integrations/SymantecCloudSOC/README.md

```diff
@@ -0,0 +1,52 @@
+Gets Events from Symantec CloudSOC.
+This integration was integrated and tested with version 3.157 of Symantec CloudSOC.
+
+## Configure Symantec Cloud SOC Event Collector on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Symantec Cloud SOC Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Server URL (e.g.<https://api.elastica.net/casb/>) | True |
+    | Key ID | True |
+    | Key Secret | True |
+    | First fetch time | False |
+    | Maximum number of incidents per fetch | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+
+Symantec CloudSOC Event Collector collects the following event types:
+* Investigate logs
+* Detect incidents logs
+  
+### API Limitations
+* You cannot retrieve investigate logs that are older than 180 days. Therefore, if setting a first fetch that is more than 180 days, for investigate logs it will be a maximum of 180 days.
```

Comment:
```suggestion
You cannot retrieve investigate logs that are older than 180 days. Therefore, if setting a first fetch that is more than 180 days, for investigate logs it will be a maximum of 180 days.
```